### PR TITLE
feat(UI): Make post listing (icon) avatars smaller (line-height)

### PR DESCRIPTION
**Pull request by jsit**
Need opinions on this. This will reduce some visual noise and also tighten things up vertically a little.

## Before

Screenshot 2023-06-25 at 4 37 21 PM

## After

Screenshot 2023-06-26 at 6 06 42 PM
Screenshot 2023-06-26 at 6 09 19 PM

### CODE REVIEW

**Review by SleeplessOne1917 — APPROVED** [review]
I think it looks better. Waiting for others' feedback before merge.

**Review by alectrocute — APPROVED** [review]
I think it looks better, too.

**Comment by jsit** [comment]
@alectrocute @SleeplessOne1917 This feels like a relatively large decision, should we wait for @dessalines to weigh in?

**Comment by alectrocute** [comment]
> @alectrocute @SleeplessOne1917 This feels like a relatively large decision, should we wait for @dessalines to weigh in?

Yep, let’s do.

**Comment by dessalines** [comment]
I am going to dissent and say I don't like it too much. What I discovered in playing around with this in jerboa, was that the icon size was actually more important in making things look cleaner: Right now, they're wayy too big in lemmy-ui. In jerboa they're properly sized to the text, and it makes a lot of difference.

post list-view: 

Screenshot_2023-06-26-15-13-23-905_com jerboa

post card-view:

Screenshot_2023-06-26-15-16-49-006_com jerboa

And it smartly hides the community icon on the community page.

The other thing is that icons, even if they're tiny, really help with instant recognition of users and communities. When you have icons in one place but not the other, its throws that off.

**Comment by jsit** [comment]
@dessalines I hear you -- glad to have gotten your feedback on this before merging. Would it be a welcome PR to make them a little smaller in the post list view?

**Comment by alectrocute** [comment]
> @dessalines I hear you -- glad to have gotten your feedback on this before merging. Would it be a welcome PR to make them a little smaller in the post list view?

We should definitely at least make the avatars line-height IMO.

**Comment by dessalines** [comment]
@jsit it'd be a good idea to make them match the text line height closely everywhere (except the tops of the profile and community pages).

**Comment by jsit** [comment]
@alectrocute @dessalines I've restored the icon size, and made the height/width equal to one line height. Updated screenshots in description.

**Review by SleeplessOne1917 — CHANGES_REQUESTED** [review]
> I've restored the icon size, and made the height/width equal to one line height. Updated screenshots in description.

I think you might have borked a merge, because only code changes I see in the PR currently is passing `hideAvatars` throught post listings.

**Comment by jsit** [comment]
@SleeplessOne1917 Oops you're right, I forgot to push. Should be good now.

File: src/shared/components/common/pictrs-image.tsx
Comment (SleeplessOne1917) [inline src/shared/components/common/pictrs-image.tsx:42]: There`s an `me-` without a number at the end of the classnames.

File: src/shared/components/post/post-listing.tsx
Comment (SleeplessOne1917) [inline src/shared/components/post/post-listing.tsx:109]: Do you have more changes you forgot to push? `smallAvatar` is defined as a prop but not passed in anywhere. I'm assuming there are also supposed to be changes to person and community listings that this prop gets passed down to.

**Review by SleeplessOne1917 — CHANGES_REQUESTED** [review]

File: src/shared/components/common/pictrs-image.tsx
Comment (jsit) [reply src/shared/components/common/pictrs-image.tsx:42]: It's a `1`; the color contrast might have gotten you here

File: src/shared/components/post/post-listing.tsx
Comment (jsit) [reply src/shared/components/post/post-listing.tsx]: Missed that one, thanks. Removed and pushed.

**Comment by alectrocute** [comment]
Super excited for this PR. Thanks @jsit

**Review by SleeplessOne1917 — APPROVED** [review]

**Pull request merged**
